build/tools.props

```diff
@@ -0,0 +1,5 @@
+<Project>
```

Comment:
We shouldn't check this file in, it should just be generated based off the global.json in the dependencies.props format.